types/api/v1beta1/event.proto

```diff
@@ -0,0 +1,162 @@
+syntax = "proto3";
+
+option go_package = "github.co/aquasecurity/tracee/api/v1beta1";
+
+package tracee.v1beta1;
+
+message Event {
+    int64 timestamp = 1;
+    int64 id = 2;
+    string name = 3;
+    // TODO: version? are we going to have version here, or just at the definition?
+    map<string, string> tags = 4; // TODO:  is this type correct?
+    map<string, string> labels = 5; // TODO:  is this type correct?
+    Policies policies = 6;
+    Context context = 7;
+    repeated EventData data = 8;
+    optional Threat threat = 9;
+}
+
+message Policies {
+    repeated string matched = 1;
+    repeated string actions = 2;
+}
+
+message Context {
+    Process process = 1;
+    optional Container container = 2;
+    optional Kubernetes kubernetes = 3;
+}
+
+message Process  {
+    int64 executionTime = 1;
+    Executable executable = 2;
+    int64 pid = 3;
+    int64 namespaced_pid = 4;
+    uint32 entity_id = 5;
+    User real_user = 6;
+    User user = 7;
+    Parent parent = 8;
+    Thread thread = 9;       
+}
+
+message Executable {
+    string name = 1;
+    string path = 2;
+}
+
+message User {
+    int64 id = 1;
+    string name = 2;
+}
+
+message Parent {
+    int64 pid = 1;
+    uint32 entityId = 2;
+    Executable executable = 3;
+    int64 start = 4;
+    optional Parent parent = 5;
+}
+
+message Thread {
+    int64 start = 1;
+    string name = 2;
+    int64 tid = 3;
+    int64 namespaced_tid = 4;
+    repeated string capabilities = 5;
+    string syscall = 6;
+    bool compat = 7;
+    repeated uint64 user_stack_strace = 8;
+}
+
+message Container {
+    string id = 1;
+    string name = 2; 
+    ContainerImage image = 3;
+    bool started = 4;
+    int64 start = 5;
+    int64 pid = 6;
```

Comment:
int32 should be enough.
Same qestion about a field which doesn't currently exist (there are some more fields like this, but I see you included them all, so I guess we want them included?)